frontend/rolecall/src/app/app/side_nav.component.scss

```diff
@@ -77,6 +77,10 @@
 
   .title {
     font-size: xx-large;
+    @media (max-width: 1200px) {
```

Comment:
This is great! 

Could you add a constant in `styles/_constants.scss` for `$breakpoint-tablet: 1200px;` so you could reuse it in all the places as `@media (max-width: $breakpoint-tablet) {`